easistrain/EDD/strainStressd0cstEDD.py

```diff
@@ -100,126 +92,387 @@ def deforDirMeasStress(anglesAndXEC, s11, s22, s33, s23, s13, s12):
-				boundsSig33 = [-np.inf , np.inf] ## bounds on sigma33
-				boundsSig23 = [-np.inf if len(pointInPeak[pointInPeak[:,10] * pointInPeak[:,11] >= 0.1, 8]) > 0 else -10**-10,
-				np.inf if len(pointInPeak[pointInPeak[:,10] * pointInPeak[:,11] >= 0.1, 8]) > 0 else 10**-10] ## bounds on sigma23
-				boundsSig13 = [-np.inf if len(pointInPeak[pointInPeak[:,9] * pointInPeak[:,11] >= 0.1, 8]) > 0 else -10**-10,
-				np.inf if len(pointInPeak[pointInPeak[:,9] * pointInPeak[:,11] >= 0.1, 8]) > 0 else 10**-10] ## bounds on sigma13
-				boundsSig12 = [-np.inf if len(pointInPeak[pointInPeak[:,9] * pointInPeak[:,10] >= 0.1, 8]) > 0 else -10**-10,
-				np.inf if len(pointInPeak[pointInPeak[:,9] * pointInPeak[:,10] >= 0.1, 8]) > 0 else 10**-10] ## bounds on sigma12
-				#print(strainTensorInitialGuess)
-				#print(stressTensorInitialGuess)
-				#print(([boundsEps11[0],boundsEps22[0],boundsEps33[0],boundsEps23[0],boundsEps13[0],boundsEps12[0]],
-				#[boundsEps11[1],boundsEps22[1],boundsEps33[1],boundsEps23[1],boundsEps13[1],boundsEps12[1]]))
-				#print(([boundsSig11[0],boundsSig22[0],boundsSig33[0],boundsSig23[0],boundsSig13[0],boundsSig12[0]],
-				#[boundsSig11[1],boundsSig22[1],boundsSig33[1],boundsSig23[1],boundsSig13[1],boundsSig12[1]]))
-				strainTensorComponents, covarStrains = scipy.optimize.curve_fit(
-				f = deforDirMeas,
-				xdata = pointInPeak[:, 3:8],
-				ydata = pointInPeak[:, 8],
-				p0 = strainTensorInitialGuess,
-				sigma = upointInPeak[:, 8],
-				bounds = ([boundsEps11[0],boundsEps22[0],boundsEps33[0],boundsEps23[0],boundsEps13[0],boundsEps12[0]],
-				[boundsEps11[1],boundsEps22[1],boundsEps33[1],boundsEps23[1],boundsEps13[1],boundsEps12[1]])
-				) ## fit of the strain tensor
-				stressTensorComponents, covarStress = scipy.optimize.curve_fit(
-				f = deforDirMeasStress,
-				xdata = np.append(pointInPeak[:, 3:8],np.array([[XEC[2 * peakNumber],XEC[2 * peakNumber + 1],0,0,0]]), axis = 0),
-				ydata = pointInPeak[:, 8],
-				p0 = stressTensorInitialGuess,
-				sigma = upointInPeak[:, 8],
-				bounds = ([boundsSig11[0],boundsSig22[0],boundsSig33[0],boundsSig23[0],boundsSig13[0],boundsSig12[0]],
-				[boundsSig11[1],boundsSig22[1],boundsSig33[1],boundsSig23[1],boundsSig13[1],boundsSig12[1]])
-				) ## fit of the stress tensor
-				pointInPeakGroup[0, 3:9] = strainTensorComponents  ## strain tensor components
-				upointInPeakGroup[0, 3:9] = np.sqrt(np.diag(covarStrains)) if np.sum(covarStrains) != np.inf else np.mean(upointInPeak[:, 8]) ## uncertainty on the strain tensor components
-				pointInPeakGroupStress[0, 3:9] = stressTensorComponents  ## stress tensor components
-				upointInPeakGroupStress[0, 3:9] = np.sqrt(np.diag(covarStress)) if np.sum(covarStress) != np.inf else (1/(XEC[2 * peakNumber + 1] + XEC[2 * peakNumber])) * np.mean(upointInPeak[:, 8]) ## uncertainty on the strain tensor components
-				
-	h5Save.close()
-	return
+    fileRead: str, fileSave: str, numberOfPeaks: int, XEC: Sequence[float]
```

Comment:
Except for type hints addition, nothing was changed. Changes are only due to `black` formatting